src/databricks/labs/ucx/queries/assessment/main/30_3_job_details.sql

```diff
@@ -1,12 +1,26 @@
-/* --title 'Job Details' --filter job_name --width 6 */
+/*
```

Comment:
actually, remove this widget altogether, because it's effectively a duplicate of `src/databricks/labs/ucx/queries/assessment/main/30_3_jobs.sql`